File: VIS360.Core/Entities/Demographic.cs
using System;
using System.Collections.Generic;
using System.Text;
using VIS360.Core.Enumerations;

namespace VIS360.Core.Entities
{
    public class Demographic
    {
        public int ID { get; set; }
        public string City { get; set; }
        public Education Education { get; set; }
        public Gender Gender { get; set; }
        public int Age { get; set; }
        public FamilyStatus FamilyStatus { get; set; }
        public Work Work { get; set; }
        public int Roommates { get; set; }
        public List<RoommateRelations> RoommateRelations { get; set; }
        public FinancialStatus FinancialStatus { get; set; }
        public List<Industries> Industries { get; set; }

        public virtual User User { get; set; }
    }
}

File: android/app/src/main/java/com/example/ikesios/Demographic.java
package com.example.ikesios;

import java.util.List;

public class Demographic {


    public int ID;
    public String City;
    public int Education;
    public int Gender;
    public int Age;
    public int FamilyStatus;
    public int Work;
    public int Roommates;
    public List<Integer> RoommateRelations;
    public int FinancialStatus;
    public List<Integer> Industries;
    public User User ;

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getCity() {
        return City;
    }

    public void setCity(String city) {
        City = city;
    }

    public int getEducation() {
        return Education;
    }

    public void setEducation(int education) {
        Education = education;
    }

    public int getGender() {
        return Gender;
    }

    public void setGender(int gender) {
        Gender = gender;
    }

    public int getAge() {
        return Age;
    }

    public void setAge(int age) {
        Age = age;
    }

    public int getFamilyStatus() {
        return FamilyStatus;
    }

    public void setFamilyStatus(int familyStatus) {
        FamilyStatus = familyStatus;
    }

    public int getWork() {
        return Work;
    }

    public void setWork(int work) {
        Work = work;
    }

    public int getRoommates() {
        return Roommates;
    }

    public void setRoommates(int roommates) {
        Roommates = roommates;
    }

    public List<Integer> getRoommateRelations() {
        return RoommateRelations;
    }

    public void setRoommateRelations(List<Integer> roommateRelations) {
        RoommateRelations = roommateRelations;
    }

    public int getFinancialStatus() {
        return FinancialStatus;
    }

    public void setFinancialStatus(int financialStatus) {
        FinancialStatus = financialStatus;
    }

    public List<Integer> getIndustries() {
        return Industries;
    }

    public void setIndustries(List<Integer> industries) {
        Industries = industries;
    }

    public User getUser() {
        return User;
    }

    public void setUser(User user) {
        User = user;
    }
}
